### ISSUE TITLE
Request textDocument/definition failed when clicking on require

### ISSUE THREAD

**Issue by jirutka** [castwide/vscode-solargraph#273]
I’m getting the following error when trying to Ctrl + click on a `require`
```
[Error - 1:34:57 PM] Request textDocument/definition failed.
  Message: [NoMethodError] undefined method `filename' for ["/home/user/project/lib"]:Array

          uri: file_to_uri(dloc.filename),
                               ^^^^^^^^^
  Code: -32603 
```

- - -
- VSCodium 1.81.1 running in Flatpak
- vscode-solargraph 0.24.0
- solargraph 0.49.0

**Comment by castwide** [comment]
This is an issue with the Solargraph gem that was reported in https://github.com/castwide/solargraph/issues/675. The fix will be included in this week's release.

**Comment by castwide** [comment]
The fix is released in gem v0.50.0. Please refer to https://github.com/castwide/solargraph/issues/675 for more information.
